admin/requirements.txt

```
@@ -19,7 +19,10 @@ jinja2>=2.11.3
 jupyter-client==6.1.7
 jupyter-core==4.7.0
 MarkupSafe==1.1.1
+matplotlib
 mccabe==0.6.1
+numpy
+pandas
 parso==0.7.1
 pickleshare==0.7.5
 prompt-toolkit==3.0.8
```

```suggestion
prompt-toolkit==3.0.8
python-doten
```

For loading the environment variables when working locally in python notebook!